### ISSUE TITLE
Cannot activate 'Select file' link in the component using keyboard.

### ISSUE THREAD

**Issue by yalagand** [adobe/spectrum-web-components#732]
A user can click on 'Select file' link to upload a file but keyboard aren't able to activate it using just keyboard. 

Also, lack of href attribute means it is not treated as a link for screenreader users.
